cmd/devtools/config/networks.go

```diff
@@ -0,0 +1,143 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/viper"
+)
+
+// NetworksConfig is the struct that holds all Astria network configurations.
+type NetworksConfig struct {
+	Networks Networks `mapstructure:"networks" toml:"networks"`
+}
+
+// Networks is the struct that holds the configuration for all individual Astria networks.
+type Networks struct {
```

Comment:
Do you need this struct? Can you just put this shape inside the above `NetworkConfigs`?

Reply:
These structs have been refactored.